src/probability/moments.lean

```diff
@@ -193,6 +205,67 @@ begin
   exact real.log_mul (mgf_pos' hμ h_int_X).ne' (mgf_pos' hμ h_int_Y).ne',
 end
 
+/-- **Chernoff bound** on the upper tail of a real random variable. -/
+lemma measure_ge_le_exp_mul_mgf [is_finite_measure μ] (ε : ℝ) (ht : 0 ≤ t)
+  (h_int : integrable (λ ω, real.exp (t * X ω)) μ) :
```

Comment:
can you `open real` in this file, and write `exp` instead of `real.exp` everywhere, or does it clash with something?